main.py

```diff
@@ -1,7 +1,30 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    for genre in genres:
+        Genre.objects.create(name=genre)
+
+    actors = ["George Klooney",
+              "Kianu Reaves",
+              "Scarlett Keegan",
+              "Will Smith",
+              "Jaden Smith",
+              "Scarlett Johansson"]
+    for actor in actors:
+        first_name, last_name = actor.split()
```

Comment:
Ensure that all actor names in the list have exactly two parts (first and last name) to avoid errors when splitting the string. If any name has more than two parts, this line will raise a ValueError.